tests: Adding a bgp router id chaos test case.

gpnaveen opened the pull request:
This is test when ospf and bgp router ids are same.

ton31337 commented:
```
Pylint report for my_frr-159185/tests/topotests/bgp_unique_rid/test_bgp_unique_rid.py:
************* Module test_bgp_unique_rid
 my_frr-159185/tests/topotests/bgp_unique_rid/test_bgp_unique_rid.py:919: error (E0602, undefined-variable, test_bgp_unique_rid_chaos2_p2) Undefined variable 'verify_ospf_neighbor'
 my_frr-159185/tests/topotests/bgp_unique_rid/test_bgp_unique_rid.py:942: error (E0602, undefined-variable, test_bgp_unique_rid_chaos2_p2) Undefined variable 'verify_ospf_neighbor'
 my_frr-159185/tests/topotests/bgp_unique_rid/test_bgp_unique_rid.py:984: error (E0602, undefined-variable, test_bgp_unique_rid_chaos2_p2) Undefined variable 'clear_ospf'
```
This must be fixed.

riw777 commented:
code looks okay, just waiting on fix for ci failure

riw777 reviewed (approved):
looks okay ... waiting on ci failure

gpnaveen commented:
> ```
> Pylint report for my_frr-159185/tests/topotests/bgp_unique_rid/test_bgp_unique_rid.py:
> ************* Module test_bgp_unique_rid
>  my_frr-159185/tests/topotests/bgp_unique_rid/test_bgp_unique_rid.py:919: error (E0602, undefined-variable, test_bgp_unique_rid_chaos2_p2) Undefined variable 'verify_ospf_neighbor'
>  my_frr-159185/tests/topotests/bgp_unique_rid/test_bgp_unique_rid.py:942: error (E0602, undefined-variable, test_bgp_unique_rid_chaos2_p2) Undefined variable 'verify_ospf_neighbor'
>  my_frr-159185/tests/topotests/bgp_unique_rid/test_bgp_unique_rid.py:984: error (E0602, undefined-variable, test_bgp_unique_rid_chaos2_p2) Undefined variable 'clear_ospf'
> ```
> 
> This must be fixed and please rebase on the latest master.

Fixed and rebased.

riw777 commented:
can we get the lint fixed on this? it's minor, I know ... but ...

gpnaveen commented:
CI: micronet

gpnaveen commented:
ci: rerun

gpnaveen commented:
CI: micronet

gpnaveen commented:
ci: rerun

riw777 commented:
unrelated ospf failure 
ci:rerun

riw777 commented:
still failing in ospf gr
ci:rerun
